ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py

```diff
@@ -884,3 +884,40 @@ def test_nextafter(
         x1=x[0],
         x2=x[1],
     )
+
+
+# diff
+@handle_test(
+    fn_tree="functional.experimental.diff",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=3,
+        min_dim_size=1,
+        max_dim_size=3,
+    ),
+    num_positional_args=helpers.num_positional_args(fn_name="diff"),
+)
+def test_diff(
```

Comment:
The check is now running for `test_diff` and this is the error message:
```
/ivy/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py:850
__ ERROR at setup of test_diff[cpu-ivy.functional.backends.torch-False-False] __
file /ivy/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py, line 890
  @handle_test(
      fn_tree="functional.experimental.diff",
      dtype_and_x=helpers.dtype_and_values(
          available_dtypes=helpers.get_dtypes("float"),
          min_num_dims=1,
          max_num_dims=3,
          min_dim_size=1,
          max_dim_size=3,
      ),
      num_positional_args=helpers.num_positional_args(fn_name="diff"),
  )
  def test_diff(
E       fixture 'container' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, include_metadata_in_junit_xml, json_metadata, metadata, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, run_around_tests, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/ivy/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py:890
```

Reply:
Perhaps take a look at the other `experimental` test functions that passed and see if you can find some differences

Reply:
Hey, @simonetgordon how to resolve this error? I tried but was not able to solve it. I also checked for other available tests but got the same error.

![Screenshot from 2022-11-20 14-01-31](https://user-images.githubusercontent.com/78868769/202892888-195dcf87-6549-417c-b319-861f99765abb.png)

Reply:
@anurag629 Hi there. I actually encountered this issue myself earlier in the week. I believe it is related to accidental changes to `.idea` folder configuration files being pushed to master recently. I tried fixing a couple of the issues in the configuration files that I noticed (e.g. typos) but it didn't seem to work. In the end, I deleted my local Ivy repo and set up again i.e. re-cloned etc. Then there were only the following files in the `.idea` directory:
```
inspectionProfiles
ivy.iml
misc.xml
modules.xml
vcs.xml
workspace.xml
```

You could either do as I did and delete your local code base and re-clone, or you could try deleting the configuration files which don't appear on the list above.

Reply:
Apologies for the issues by the way, let me know how you progress with fixing it!

In the meantime, you can still inspect the output of the test function via the details of the PR check i.e. `test-experimental-core-ivy` -> `numpy, elementwise` -> 'Run tests' and search the error output for `test_diff`

Reply:
Hi @simonetgordon Now tests are failing due to the zeta function.

https://github.com/unifyai/ivy/actions/runs/3510006434/jobs/5879494232#step:7:59

![Screenshot from 2022-11-21 03-55-11](https://user-images.githubusercontent.com/78868769/202929718-7f8f02a6-97c1-4140-8562-999fd3e8702e.png)